app/models/grda_warehouse/service_history_enrollment.rb

```diff
@@ -4,6 +4,10 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# = GrdaWarehouse::ServiceHistoryEnrollment
+#
+# ServiceHistoryEnrollments flatten Hud Enrollments and related records to serve reporting needs. These records are
+# generated automatically. There is a 1:1 correspondence with Hud Enrollment records
```

Comment:
This is only partially true.  There should be a ServiceHistoryEnrollment with a `record_type=entry` that corresponds to the Enrollment record, one with `record_type=exit` that corresponds to the Exit (if the client has an Exit record) and potentially one with `record_type=first` if the enrollment is the client's first homeless enrollment.  So there is a 1:1 with Enrollment for `GrdaWarehouse::ServiceHistoryEnrollment.entry`